src/posthog-core.ts

```diff
@@ -296,6 +300,20 @@ export class PostHog {
         this._retryQueue = undefined as any
         this.persistence = undefined as any
         this.sessionManager = undefined as any
+
+        // NOTE: See the property definition for deprecation notice
+        this.people = {
+            set: (prop: string | Properties, to?: string, callback?: RequestCallback) => {
+                const setProps = typeof prop === 'string' ? { [prop]: to } : prop
+                this.identify(this.get_distinct_id(), setProps)
```

Comment:
I would prefer if we use `$set` events here - can be a function `setUserProperties` or whatever  that takes set and setOnce properties (doesn't need to be the complex person file). Why: `identify` is complex both here and in plugin-server and  hence is more likely to have side-effects and bugs.